way_to_home/way/models.py

```diff
@@ -1 +1,38 @@
 """This module implements class that represents the way entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
```

Comment:
```suggestion
from user.models import User
from django.db import models, IntegrityError
from utils.abstract_models import AbstractModel
```

Optimize imports, please